bin/show_org_stats.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :since, "Since what date.", :type => :string, :required => true
+
+  ManageIQ::Release.common_options(self, :except => :dry_run)
+end
+
+class OrgStats
+  def initialize(since:, **opts)
+    @since = since
+    @opts  = opts
+
+    @total_commits = Hash.new(0)
+    @total_merges  = Hash.new(0)
+    @names = {}
+  end
+
+  def run
+    ManageIQ::Release.each_repo(**@opts) { |repo| run_one(repo) }
+
+    puts
+    puts "Total Commits:"
+    print_totals(@total_commits)
+
+    puts
+    puts "Grand Total Commits: #{@total_commits.values.sum}"
```

Comment:
not necessarily for this PR, but would be nice to add number of different PR authors

Reply:
I updated the OP with example output

Reply:
thanks!

Reply:
is this counting commits or PRs?  Same question for the merge counts.

Reply:
The first half is commits, but not merges...the second half is merges, but not commits.  None of these are github stats; they are git stats.

Reply:
I can rename to show_org_git_stats.rb if that makes it clearer?